docs/error_codes.md

```diff
@@ -8,8 +8,7 @@ Threshold is `equal to zero`, some error could happen during the Safe setup.
 
 ### General gas/ execution related
 - **GS010: Not enough gas to execute Safe transaction**  
-SafeTxHash is too high or gas limit is too low to execute the transaction, keep in mind that `2500 gwei` are needed to events and `500 gwei` to perform code until execution.  
-Review this values and increase the `gasLimit` or reduce the `safeTxHash` according the current gas price and your requirements.  
+SafeTxGas is too high or gas limit is too low to execute the transaction, keep in mind that `2500 gwei` are needed to events and `500 gwei` to perform code until execution.  
```

Comment:
- `are needed for events`
- `at least 500`, as if you have contract signatures it can be more due to calls to another contracts
- gwei?